### ISSUE TITLE
Fix audio downloader

### ISSUE THREAD

**Issue by Ch3ck** [Ch3ck/youtube-dl#15]
Current implement does not properly download the mp3 format of the youtube video

**Comment by ghost64** [comment]
Unless I am missing something goav still needs ffmpeg to convert to MP3.  Might be worth integrating it to all conversion between any format.

**Comment by Ch3ck** [comment]
@ghost64 Yeah, I was just thinking of ways to reduce dependency issues in the application. I was looking at using liblamemp3: https://github.com/viert/lame instead. But to give flexibility to the application. `ffmpeg` will be the best option.
